packages/ui/src/components/FileUpload/utils/getDisplaySize/index.ts

```diff
@@ -0,0 +1,28 @@
+/** *******************************************************************************************************************
+  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+  
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+  
+      http://www.apache.org/licenses/LICENSE-2.0
+  
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.                                                                              *
+ ******************************************************************************************************************** */
+const getDisplaySize = (size?: number | null) => {
+    if (!size) return undefined;
+
+    const k = 1000;
+    const dm = 2;
+    const sizes = ['bytes', 'KB', 'MB', 'GB'];
+
+    const i = Math.floor(Math.log(size) / Math.log(k));
+
+    return `Size: ${parseFloat((size / Math.pow(k, i)).toFixed(dm))} ${sizes[i]}`;
```

Comment:
i guess that `parseFloat` is unnecessary here
```suggestion
    return `Size: ${(size / Math.pow(k, i)).toFixed(dm)} ${sizes[i]}`;
```